Assets/Events/ExcessEvents.json

```
@@ -0,0 +1,67 @@
+[
+  {
+    "name": "brawler-excess",
+    "headline": "Brawlers Overjoyed!",
+    "article": "Brawlers are so satisfied by their current living conditions, and the word has spread! 5 new brawlers have joined the town this morning, ready to punch anything and everything.",
```

For all of these "conditions **that** the word has spread"
```suggestion
    "article": "Brawlers are so satisfied by their current living conditions that the word has spread! 5 new brawlers have joined the town this morning, ready to punch anything and everything.",
```